src/job-posting/job.service.ts

```
@@ -272,4 +280,174 @@ export class JobService {
+    return updatedJob;
+  }
+
+  // Auto-release payment after review period expires
+  @Cron(CronExpression.EVERY_HOUR) // Check every hour
+  async autoReleasePayments(): Promise<void> {
+    const now = new Date();
+
+    const jobsToAutoRelease = await this.jobRepository.find({
+      where: {
+        completionStatus: CompletionStatus.PENDING_REVIEW,
+        reviewDeadline: require('typeorm').LessThanOrEqual(now),
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)